salt/renderers/gpg.py

```diff
@@ -37,6 +37,55 @@
     increasing the system entropy. On virtualised Linux systems, this can often
     be achieved by installing the ``rng-tools`` package.
 
+Import keys to a master
+************************
+
+If the keys already exist and need to be imported to the salt master, run the
```

Comment:
It looks like we're assuming the key name here, aren't we? We should perhaps clarify the naming so that people don't just blindly run these commands and then get confused when they don't work.

Reply:
Hmm, the example shows some filenames that may differ from case to case. One can choose the filename during export... Ideas how to make this more precise?

Reply:
Changed key imort example to use `/path/to/key` syntax instead of just a filename.